src/main/resources/default/taglib/t/blobImageHardRefField.html.pasta

```diff
@@ -76,6 +76,38 @@
     }
     Dropzone.options[sirius.camelize('@id')].init = function () {
         const dropzone = this;
+        let _dropzoneIndicator = document.querySelector('#' + '@id' + ' .sirius-upload-hover');
```

Comment:
Can't this block kind reuse https://github.com/scireum/sirius-web/pull/1035/commits/4521fc408a3f1b37d70b3fdf0840720a08505ad1?

Reply:
its not that simple to extend the code used in web here, and the additional stuff for the image preview doesn't really make sense to be added to the base field